examples/cross-browser.py

```diff
@@ -84,11 +85,13 @@ async def main():
     # Puppeteer:
     # await page.goto('https://news.ycombinator.com/');
```

Comment:
please update the comment. And maybe add clarification where the page came from.

Reply:
Done down below